media/css/firefox/family/components/modules/_public-wifi.scss

```diff
@@ -26,7 +27,7 @@
     }
 
     .c-card {
-        margin-bottom: $layout-2xs;
+        margin: $layout-2xs;
```

Comment:
**issue**: card spacing looks uneven
<img width="988" alt="Screen Shot 2022-09-23 at 11 23 38 AM" src="https://user-images.githubusercontent.com/19650432/191941263-5acbe240-08ce-41cd-94c9-47d902242a14.png">

**suggestion**: we can simplify by removing the `max-width: $screen-lg styles` from L38-60. These were added to allow the scrolled content to go to the edge of the screen (so mzp-l-content padding had to be removed). Since we don't need the scroll, we can restore the usual mzp-l-content padding by removing this override. We should be able to remove card margin on L17 and L30 too, and restore the margin-bottom style to keep vertical space between the yellow and white cards.